### ISSUE TITLE
Mathfield prevents scrolling of page

### ISSUE THREAD

**Issue by stefnotch** [arnog/mathlive#1146]
## Description

When hovering over a mathfield with the mouse, the scroll wheel will not have an effect.
This used to be quite noticeable in my and phcreery's [3blue1brown contest entry](https://stefnotch.github.io/quantum-sheet/?document-url=https://gist.githubusercontent.com/stefnotch/fe8a80a57d2ee9211b7a24ab7fcc9ce2/raw/document.qs), but for now I fixed it by locally patching mathlive and deploying that.

## Steps to Reproduce

1. Go to https://cortexjs.io/compute-engine/demo/
2. Move the mouse over the mathfield
3. Use the scroll wheel. Notice how nothing happens

### Actual Behavior

No scrolling happens.

### Expected Behavior

I expected the page to get scrolled. Or alternatively, the mathfield to get scrolled to the top/bottom and then the page scrolling.

### Environment

**MathLive version** v0.69.5

**Operating System** Windows 10

**Browser** Firefox 91

**Comment by arnog** [comment]
Could you share your patch?

**Comment by stefnotch** [comment]
> 
> 
> Could you share your patch?

Of course. I opted for the laziest option, which was to just remove the 'scrolling preventing' part and potentially break any equations that would lead to scrolling being possible. Thus, I removed those two lines
https://github.com/arnog/mathlive/blob/8d3816f01118c1a198ef7d73c64b66218bbe00eb/src/editor-mathfield/mathfield-private.ts#L336-L337

I suppose a more proper fix would be to first check if the mathfield can be scrolled or if we're at the end. And only if we can scroll the mathfield, the scroll event should be stopped. 

On that note, I just noticed that this event handler seems to be borked? I couldn't get it to do anything on Firefox and Microsoft Edge.
